code/modules/clothing/suits/armor.dm

```diff
@@ -77,17 +77,18 @@
 	item_state = "capspacesuit"
 
 /obj/item/clothing/suit/armor/riot
-	name = "riot suit"
-	desc = "A suit of armor with heavy padding to protect against melee attacks. Looks like it might impair movement."
+	name = "EOD suit"
+	desc = "A suit of armor with heavy padding to protect against most attacks. Looks like it might impair movement."
 	icon_state = "riot"
 	item_state = "swat_suit"
 	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
 	cold_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
 	heat_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
-	armor = list(melee = 50, bullet = 10, laser = 10, energy = 10, bomb = 0, bio = 0, rad = 0)
+	armor = list(melee = 60, bullet = 80, laser = 40, energy = 30, bomb = 100, bio = 50, rad = 0)
```

Comment:
I am not so sure about this 'EOD' suit, especially when sec already has a suit for bombs. If you wanted to create some sort of tank or bulldozer armor, you should do it with a new suit and path! Balance the riot armor as it currently is, a suit to counter melee attacks.

**Chaplain's knight armor is a child of riot armor!**